### ISSUE TITLE
[ui] UI needs to implement a logout page

### ISSUE THREAD

**Issue by Tomcli** [machine-learning-exchange/mlx#78]
**Is your feature request related to a problem? Please describe.**
Right now there's no way for the admin to logout other than clear up the browser cookies, so we need a logout page to simply this process.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by ckadner** [comment]
I don't think we need a new page. Just a logout button should suffice. Or, as a quick(er)-fix, we could just implement a "`Route`" (?) instead of a whole page. Meaning, if a logged-in admin user appends "/logout" in the browser's URL to trigger the logout.
